solutions/mint-nft/utils/constant.ts

```diff
@@ -0,0 +1,3 @@
+const NFTAddress = "0x343280B77B62F0DC48243dDFE2D4bC7e4B1f1172";
```

Comment:
Is this a needed placeholder?

Reply:
Yes we need to, in order to mint the NFT, which means writing to the smart contract, we'll need to specify which contract address to write to